packages/game/src/ts/frontend/spaceship/spaceship.ts

```diff
@@ -584,6 +592,17 @@ export class Spaceship implements Transformable {
 
         const warpDrive = this.getInternals().getWarpDrive();
 
+        // Low fuel warning check
+        const fuelPercentage = this.getRemainingFuel() / this.getTotalFuelCapacity();
+        if (fuelPercentage < 0.2 && fuelPercentage > 0 && !this.lowFuelWarningTriggered) {
```

Comment:
`0.2` should be a variable instead of a literal since it is used twice (maybe the value will change in the future)